src/components/UserProfile/UserSettings/UserNotificationSettings/index.tsx

```diff
@@ -97,7 +111,46 @@ const UserNotificationSettings: React.FC = () => {
               </div>
               <div className="form-row">
                 <label htmlFor="discordId" className="text-label">
-                  {intl.formatMessage(messages.discordId)}
+                  <span>{intl.formatMessage(messages.discordId)}</span>
+                  <span className="label-tip">
+                    {intl.formatMessage(messages.discordIdTip, {
+                      FindDiscordIdLink: function FindDiscordIdLink(msg) {
+                        const discordSupportLocales = [
+                          'da',
+                          'de',
+                          'es',
+                          'fr',
+                          'it',
+                          'ja',
+                          'ko',
+                          'nl',
+                          'pl',
+                          'pt-br',
+                          'ru',
+                          'sv',
+                          'tr',
+                          'zh-tw',
+                        ];
```

Comment:
This is really thoughtful but also not something we should maintain.

Reply:
Is there a better way to do this?  Ideally the Discord support website would just redirect if an invalid language code is used, but it just fails.  NBD if we want to just link to the English article, I suppose.

Reply:
Not that I am aware of. It's on discord to handle it though.

Reply:
Okay, will remove this.